### DISCUSSION TITLE
Local geotiff to be served and ability to have RGB composite on the fly.

### DISCUSSION THREAD

**Discussion by U-Tyagi** [developmentseed/titiler#1045] [Q&A]
Q1. I need to serve tiles through Titiler just as a tile server for a geotiff stored locally. Can someone share any example of doing this.
Q2. I have multiple bands and need to have tiles served of RGB composites combing 3 bands together.

Any pointers.?

**Comment by vincentsarago** [comment]
@U-Tyagi have you looked at the documentation? 

you can also have a look at https://github.com/developmentseed/rio-viz

**Comment by AndrewAnnex** [comment]
1. titiler can seemingly be provided any path on the filesystem of the server via the url. I do wonder if there are security implications for this, but anyways.
2. the bidx and expression parameters to the url work for this
